purchase_last_price_info/models/product_product.py

```diff
@@ -12,7 +12,8 @@ class ProductProduct(models.Model):
     last_purchase_line_ids = fields.One2many(
         comodel_name="purchase.order.line",
         inverse_name="product_id",
-        domain=[("state", "in", ["purchase", "done"])],
+        domain="[('state', 'in', ['purchase', 'done']),"
+        "('company_id', 'in', company_ids)]",
```

Comment:
Why is this needed?
Isn't it handled by `purchase.order.line` ir rules?

Reply:
Indeed, but It doesn't look so, or we would not see the bug.

Reply:
that solution isn't work for me...

this solution work fine:
domain=lambda self: [('state', 'in', ['purchase', 'done']),
                             ('company_id', '=', self.env.company.id)]